tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -91,7 +91,7 @@
 public class TonyClient implements AutoCloseable {
   private static final Log LOG = LogFactory.getLog(TonyClient.class);
 
-  private static final String APP_TYPE = "TENSORFLOW";
+  private static final String APP_TYPE = "TonY";
```

Comment:
TONY ?

Reply:
Looks like MAPREDUCE and SPARK use ALL_CAPS, too. Will change to TONY.